### PR TITLE
[4.0] Remove comments re IE11 from core code

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # .
#26123
### Summary of Changes
Removed IE11 support per #26160 

### Testing Instructions
1) Create or view a category blog.
    View Source
    blog-item-content class should be gone.

2) Create or view a featured article.
    View Source
    blog-item-content class should be gone.

### Expected result
image

### Actual result
image

### CODE REVIEW

**Comment by C-Lodder** [comment]
Why have you added custom ordering rules?

**Comment by N6REJ** [comment]
> Why have you added custom ordering rules?

FACEPALM! thats NOT supposed to be there *(&()(*$@#$_
How can I fix that now?

**Comment by C-Lodder** [comment]
Remove it? :D

**Comment by N6REJ** [comment]
won't that delete it in core?

**Comment by C-Lodder** [comment]
It was never there in core as far as I remember. So just remove those changes

**Comment by N6REJ** [comment]
@C-Lodder WHEW!! just had to undo the changes and re-commit it!  TY!

File: components/com_content/tmpl/category/blog.php [hunk 1]
```diff
@@ -98,12 +96,10 @@
 		<?php foreach ($this->intro_items as $key => &$item) : ?>
 			<div class="com-content-category-blog__item blog-item"
 				itemprop="blogPost" itemscope itemtype="https://schema.org/BlogPosting">
-				<div class="blog-item-content"><!-- Double divs required for IE11 grid fallback -->
-					<?php
+				<?php
```
Comment (Quy) [inline components/com_content/tmpl/category/blog.php]:
```suggestion
					<?php
```

**Comment by jduerscheid** [comment]
I have tested this item :white_check_mark: successfully on 9cea4a27407863cfbe866d8f4164b9d492d934cf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26263">issues.joomla.org/tracker/joomla-cms/26263</a>.</sub>

**Comment by joeykrop** [comment]
I have tested this item :white_check_mark: successfully on 9cea4a27407863cfbe866d8f4164b9d492d934cf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26263">issues.joomla.org/tracker/joomla-cms/26263</a>.</sub>

**Comment by crommie** [comment]
I have tested this item :white_check_mark: successfully on 9cea4a27407863cfbe866d8f4164b9d492d934cf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26263">issues.joomla.org/tracker/joomla-cms/26263</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26263">issues.joomla.org/tracker/joomla-cms/26263</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by N6REJ** [comment]
@Scrabble96 finally got it merged for you 👯‍♂

**Comment by Scrabble96** [comment]
Hooray! Well done @N6REJ

**Pull request merged**